README.md

```diff
@@ -1,7 +1,38 @@
-# Panther CLI Tool
-`panther-cli` is a Python command line interface for testing and packaging Panther Policies and Rules. Policies define the compliant and secure state of a cloud Resource, whereas Rules perform analysis on log data.
+# Panther Analysis
 
-## Installation
+This repository is a combination of open source policies and rules meant to be run with [Panther](https://github.com/panther-labs/panther), as well as the `panther-cli` tool which can be used to test and package these policies and rules without having to go through the Panther web interface.
```

Comment:
Can we mention something about the artifacts of the releases in this repo are bundled in Panther deployments by default?

Reply:
Will do, added a brief paragraph explaining the relationship.

Reply:
This repository contains open-source Panther policies and rules, along with a CLI tool for testing and packaging.